tests/unit/examples/test_building_deploying_multi_stage_RecSys.py

```diff
@@ -63,20 +60,26 @@ def test_func():
         top_k = tb2.ref("top_k")
         outputs = tb2.ref("outputs")
         assert outputs[0] == "ordered_ids"
-
-        df_lib = get_lib()
-
-        # read in data for request
-        batch = df_lib.read_parquet(
-            os.path.join("/tmp/data/processed/retrieval/", "train", "part_0.parquet"),
-            num_rows=1,
-            columns=["user_id"],
-        )
-        configure_tensorflow()
-
-        response = run_ensemble_on_tritonserver(
-            "/tmp/examples/poc_ensemble/", outputs, batch, "ensemble_model"
+        tb2.inject(
+            """
+            import shutil
+            from merlin.core.dispatch import get_lib
+            from merlin.models.loader.tf_utils import configure_tensorflow
+            configure_tensorflow()
+            df_lib = get_lib()
+            batch = df_lib.read_parquet(
```

Comment:
I am changing the notebooks in this PR  https://github.com/NVIDIA-Merlin/Merlin/pull/618 to return the raw ids. so feeding this batch wont work.  Either we can merge my PR first and then update unit test accordingly, or we can merge this PR, then merge my PR and update unit test again :)

Reply:
I think we should get what we had working, then you can modify as you see fit. :)